draft-ietf-quic-transport.md

```diff
@@ -2508,11 +2514,11 @@ transport parameter in the TLS handshake.
 Servers MAY communicate a preferred address of each address family (IPv4 and
 IPv6) to allow clients to pick the one most suited to their network attachment.
 
-Once the handshake is confirmed, the client SHOULD select one of the two
-server's preferred addresses and initiate path validation (see
+Once the handshake is confirmed, the client SHOULD select one of the two values
+preferred addresses provided by the server and initiate path validation (see
```

Comment:
Redundancy here:

```suggestion
Once the handshake is confirmed, the client SHOULD select one of the two
preferred addresses provided by the server and initiate path validation (see
```

Reply:
This is what the text already said, but I think its confusing. My suggestion is "values for preferred address provided by the server